HW1/P3/P3.txt

```diff
@@ -1 +1,3 @@
-# Your discussion here
+**Q3**
+Entry from the end RDD which has maximum number of anagrams
+[(u'aerst', 11, [u'arets', u'aster', u'rates', u'reast', u'resat', u'stare', u'stear', u'strae', u'tares', u'tears', u'teras'])]
```

Comment:
If you print the first few lists, you could see there are actually 3 lists with 11 words.